wrappers/wrapper.rb

```diff
@@ -671,7 +671,8 @@ def check_timewindow_inconsistency(vrp, unfeasible, service)
 
         max_earliest_arrival = 0
         relation.linked_services.map{ |service_in|
-          next unless service_in.activity.timewindows.any?
+          # TODO: Should consider alternatives
```

Comment:
can you add a log warning as well? So that we have a way to see how many times we ignore activities.